src/pages/organize/[orgId]/campaigns/[campId]/index.tsx

```diff
@@ -142,7 +142,9 @@ const CampaignSummaryPage: PageWithLayout<CampaignCalendarPageProps> = ({ orgId,
                 </Box>
                 <Box display="flex" flex={ 1 } p={ 1 }>
                     <Box p={ 1 }>
-                        <Avatar></Avatar>
+                        <Avatar
+                            { ...(campaign?.manager && { src: `/api/orgs/${orgId}/people/${campaign?.manager.id}/avatar` }) }>
```

Comment:
I'm not a big fan of this way of adding conditional properties, but I'm open to being convinced. Would love to hear your thoughts and motivations @nehahirve. Also, what do you think @djbusstop and @niklasva82?

I think I would prefer one of the following ways instead.

## Conditional value only (inline)
```tsx
<Avatar src={ campaign?.manager?
  `/api/orgs/${orgId}/people/${campaign?.manager.id}/avatar` : undefined }
/>
```

## Conditional value only (declared separately)
```tsx
// Higher up
const avatarSrc = campaign?.manager?
  `/api/orgs/${orgId}/people/${campaign?.manager.id}/avatar` : undefined;

// TSX
<Avatar src={ avatarSrc }/>
```

## Conditional component
```tsx
{ campaign?.manager? (
  <Avatar src={ `/api/orgs/${orgId}/people/${campaign?.manager.id}/avatar` }/>
) : <Avatar /> }
```

Reply:
There's no special advantage to doing it my way, I guess it's less readable, will change

Reply:
Honestly, I would usually do something like this:

```tsx
// Higher up
let avatar = <Avatar />;
if (campaign?.manager) {
  avatar = <Avatar src={ `/api/orgs/${orgId}/people/${campaign?.manager.id}/avatar` }/>;
}

// TSX
{ avatar }
```

But that's because I started writing JSX before we as a coding community started to gravitate towards in-JSX ternaries and the `{ someBool && <SomeComponent/> }` construct. So maybe I'm just being conservative. 😊